src/DotNetWorker.Grpc/BindingData.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Functions.Worker.Core;
+
+namespace Microsoft.Azure.Functions.Worker.Grpc.Messages
+{
+    internal sealed partial class BindingData : IBindingData
```

Comment:
For the naming of the types that are consumed by worker converters, I wonder if ParameterBindingData wouldn't be better here?

Reply:
Renaming to ModelBindingData